src/app.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+//form aanspreken en eventlistener op geplaatst. Als er wordt gesubmit/geentered, dan functie fetchcountries uitvoeren.
+const formElement = document.getElementById('receivingInput');
+formElement.addEventListener('submit', fetchCountries);
+
+//VRAAG: waarom verdwijnt de data zo snel? :(
+
+//variabele aangemaakt voor het laatste gedeelte van de uri.
+const INPUT = document.getElementById('inputfield')
```

Comment:
Variabelen camelCase schrijven
```suggestion
const input = document.getElementById('inputfield')
```